changedetectionio/fetch_site_status.py

```diff
@@ -104,9 +104,16 @@ def run(self, uuid):
             # https://stackoverflow.com/questions/41817578/basic-method-chaining ?
             # return content().textfilter().jsonextract().checksumcompare() ?
 
-            is_html = True
+            is_json = fetcher.headers.get('Content-Type', '') == 'application/json'
+            is_html = not is_json
```

Comment:
So what you're saying here, is that it should switch automatically to JSON parsing if the content type is JSON?

Reply:
Yes exactly otherwise HTML inside the JSON response will get removed by inscriptis here: https://github.com/dgtlmoon/changedetection.io/blob/f62357aeed6805e061cff14983b7ddfe9f08fa08/changedetectionio/fetch_site_status.py#L131

Reply:
the test would be, pass JSON content and then expect it to handle it just like regular content